templates/App/Datum/_datum.html.twig

```diff
@@ -1,6 +1,6 @@
 {% set hasChoiceList = choiceList is defined and choiceList is not null %}
 
-<div {% if template is defined %} data-template="{{ template.name }}"{% endif %} class="datum row" data-additional-data-target="datum" data-iteration="{{ iteration }}">
+<div {% if template is defined %} data-template="{{ template.name }}"{% endif %} class="datum row" data-additional-data-target="datum" data-iteration="{{ iteration }}" style="margin-bottom: 30px;">
```

Comment:
Please consideer creating a CSS class, because this is the only place where inline CSS are put.

Reply:
Done